#include <cstdlib>

int main() {
    std::exit(EXIT_FAILURE);
}